kickstart/scripts/samba-deploy.sh

```diff
@@ -11,6 +11,18 @@ deploy_samba_ubuntu() {
   chown nobody:nogroup /opt/data/fieldpapers
   chmod 777 /opt/data/fieldpapers
 
+  mkdir -p /opt/data/backups/omk
+  chown nobody:nogroup /opt/data/backups/omk
+  chmod 777 /opt/data/backups/omk
+
+  mkdir -p /opt/data/backups/osm
+  chown nobody:nogroup /opt/data/backups/osm
+  chmod 777 /opt/data/backups/osm
+
+  mkdir -p /opt/data/backups/fieldpapers
+  chown nobody:nogroup /opt/data/backups/osm
+  chmod 777 /opt/data/backups/osm
+
```

Comment:
Actually, rather than moving these to `admin-deploy.sh`, move them to each of the components that they belong to, i.e. `/opt/data/backups/fieldpapers` to `fieldpapers-deploy.sh`. We're then set up to add appropriate cron jobs and allow each component to manage its (automated) backup strategy.